### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Marcraphael12**
check with others if the footer is mobile, the form respects the UX, there are buttons interactions, the form action to get user information works well.
My code is yours, kindly leave a comment or visit the issue page if needed. I would be happy to apply modifications.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Marcraphael12, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -214,5 +214,45 @@ <h3>Multi-Post Stories</h3>
...
+				<label for="email">Email</label>
+				<input type="email" name="email" class="email name-email" id="email" required placeholder="email@email.com">
+			</div>
+
+			<textarea name="message" class="message" maxlength="500" placeholder="Enter text here."></textarea>
+
+			<button type="submit" class="see-project">
+				see project
+			</button>
+		</form>
```
Comment (mke2111) [inline index.html:243]: - [ ] The code in the index.html file has a different webpage from that you have in the live demo.
Here is your live demo version.
Screenshot from 2021-08-17 15-46-18


And here is the webpage based on your code.
Screenshot from 2021-08-17 15-45-50

- Fix this issue and let's stick to the right version.

File: index.html [hunk 2]
```diff
@@ -214,5 +214,45 @@ <h3>Multi-Post Stories</h3>
...
+				<label for="name">Name</label>
+				<input type="text" name="name" class="name name-email" id="name" maxlength="30" required placeholder="Enter your name">
+	
+				<label for="email">Email</label>
+				<input type="email" name="email" class="email name-email" id="email" required placeholder="email@email.com">
+			</div>
+
+			<textarea name="message" class="message" maxlength="500" placeholder="Enter text here."></textarea>
+
+			<button type="submit" class="see-project">
```
Comment (mke2111) [inline index.html:240]: - [ ] With the original webpage(index file), on submitting the form, the contents of form fields are not submitted successfully. Give this a fix

File: index.html [hunk 3]
```diff
@@ -214,5 +214,45 @@ <h3>Multi-Post Stories</h3>
...
+				If you have an application you are interested in developing, 
+				a feature that you need built or a project that needs coding. I’d love to help with it 
+			</p>
+		</div>
+
+		<!-- The form -->
+		<form  action="https://formspree.io/f/mnqlvaya"	method="POST">
+			<div>
+				<label for="name">Name</label>
+				<input type="text" name="name" class="name name-email" id="name" maxlength="30" required placeholder="Enter your name">
```
Comment (mke2111) [inline index.html:232]: - [ ] For all input fields, add the `required` attribute to activate the validation on submit

Comment (Marcraphael12) [reply index.html:232]: done thanks!

File: index.html (hunk 2 above)
Comment (Marcraphael12) [reply index.html:240]: done, thanks!

File: index.html (hunk 1 above)
Comment (Marcraphael12) [reply index.html:243]: done, thanks, even this is not a project requierement!

**Review by Murodjon000 — APPROVED** [review]
**Hi Marc Raphael 👋,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Murodjon000 — COMMENTED** [review]
**Hi Marc Raphael 👋,**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**